cmd/master.go

```diff
@@ -34,7 +34,7 @@ var masterCmd = &cobra.Command{
 		}
 
 		filename := loadConfig(check.MASTER, bv)
-		runChecks(check.MASTER, filename)
+		runChecks(check.MASTER, filename, !isMaster())
```

Comment:
This should always attempt to run regardless of node type, because the user has explicitly asked for the master tests